### ISSUE TITLE
can't override event render setting with event-source or resource

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#4420]
The `rendering` property on [Event objects](https://fullcalendar.io/docs/v4/event-parsing) can cause an event to render as a background event.

It used to be possible to specify `rendering` on the Event Source to have this apply to all events from the source.

It also used to be possible to specify `eventRendering` on a Resource to have this apply to all events associated with a resource.

The latter two features no longer work.

**Comment by baileyandy** [comment]
I'm stuck on `v4.0.0-alpha.2` because of these regressions. Are there any known workarounds please?

**Comment by baileyandy** [comment]
I ended up setting `rendering` and `editable` on each `Event` in a success callback rather than the `EventSource`, allowing an upgrade to the latest fullcalendar release

```JavaScript
this.fullCalendar.addEventSource({
//    rendering: 'background',
    events: this.getEvents,
    success: Calendar.transformToEvents
});


static transformToEvents(source: Source[]): EventInput[] {
    return source.map(_ => ({
        rendering: 'background'
    }));
};
```

with a similar fix for `editable: true` on another source.

This approach also fixes my example problems on #4345

**Comment by arshaw** [comment]
this should be fixed in v5

ps- the `rendering` setting has been renamed to `display`
